app/controllers/concerns/price_group_members_controller.rb

```diff
@@ -23,6 +23,8 @@ def initialize
   end
 
   def create
+    raise "Permission Denied" unless @price_group.can_manage_price_group_members?
```

Comment:
What is a more sensible thing to raise than this? Or should we redirect back with an error message?

Reply:
I would say raise `NUCore::PermissionDenied`. That'll render a 403. They shouldn't be able to get here otherwise.